README.md

```diff
@@ -52,7 +52,7 @@ cd front-end-monorepo
 docker-compose build
 ```
 
-`docker-compose up` runs production builds of the project app at http://localhost:3000 and the content pages app at http://localhost:3001
+`docker-compose up` runs local builds of the project app at http://localhost:3000 and the content pages app at http://localhost:3001
```

Comment:
```suggestion
`docker-compose up` runs local production builds of the project app at http://localhost:3000 and the content pages app at http://localhost:3001
```

I think we need to be clear here that we're running production builds and not dev builds.